app/src/main/java/com/github/swent/echo/compose/authentication/RegisterScreen.kt

```diff
@@ -48,8 +51,10 @@ fun RegisterScreen(registerViewModel: RegisterViewModel, navActions: NavigationA
         }
     }
 
+    val scrollState = rememberScrollState()
+
     Column(
-        modifier = Modifier.testTag("register-screen"),
+        modifier = Modifier.testTag("register-screen").fillMaxHeight().verticalScroll(scrollState),
```

Comment:
It is generally preferable to use a LazyColumn if you want a scrollable Column, but because you have only a few items, the difference in performance is low. It's not so important in this context but think about it the next time ;) 
source : https://stackoverflow.com/questions/73734918/android-compose-difference-between-lazycolumn-and-column-with-verticalscroll#:~:text=A%20LazyColumn%20is%20a%20vertically,the%20classic%20Android%20View%20system.&text=A%20Column%20will%20show%20each,a%20LinearLayout%20with%20vertical%20orientation.